addons/mail/static/src/components/message_author_prefix/message_author_prefix.xml

```diff
@@ -7,7 +7,7 @@
                 <t t-if="message.author and message.author === env.messaging.currentPartner">
                     <i class="o_MessageAuthorPrefixIcon fa fa-mail-reply"/>You:
                 </t>
-                <t t-elif="thread and thread.channel_type === 'channel'">
+                <t t-elif="thread and message.author != thread.correspondent">
```

Comment:
`!==`, even if Qweb actually treats `!=` as alias for `!==` in the JS version.

Reply:
My bad, that was a typo, didn't even know QWeb converted them.